### DISCUSSION TITLE
Ambiente di test

### DISCUSSION THREAD

**Discussion by Alexxia5** [fattureincloud/api#433] [Q&A] [answered]
Ciao. 
Volevo sapere se esiste un ambiente di test per fare sviluppo. 
Ho cercato nei post e ne ho trovato uno un pò datato in cui si diceva che non c'era un ambiente dedicato, ma che si può utilizzare un account con un mese di prova gratuita e chiedere al supporto clienti se si può prorogare per continuare a sviluppare.
Mi confermate che la situazione è ancora così oppure se nel frattempo è stato creato un ambiente dedicato agli sviluppi?
Grazie.

**Comment by valmoz** [comment, marked as answer]
Buongiorno @Alexxia5 
purtroppo al momento non sono ancora stati fatti passi avanti a riguardo.
Confermo che è possibile usare un account di trial a scopo di sviluppo, come spiegato [qui](https://developers.fattureincloud.it/docs/developer-account)

**Reply by Alexxia5** [reply]
Va bene
Grazie per la risposta.
